integration-tests/src/test/java/oracle/weblogic/kubernetes/ItKubernetesEvents.java

```diff
@@ -515,76 +512,242 @@ public void testDomainK8SEventsDelete() {
   }
 
   /**
-   * Operator logs a NamespaceWatchingStarted event in the respective domain name space
-   * when it starts watching a new domain namespace.
-   * The test upgrades the operator instance through helm to add another domain namespace in the operator watch list.
+   * Test verifies the operator logs a NamespaceWatchingStarted event in the respective domain namespace
+   * when it starts watching a new domain namespace with domainNamespaceSelectionStrategy default to List and
+   * operator logs a NamespaceWatchingStopped event in the respective domain namespace
+   * when it stops watching a domain namespace.
+   * The test upgrades the operator instance through helm to add or remove another domain namespace
+   * in the operator watch list.
+   * This is a parameterized test with enableClusterRoleBinding set to either true or false.
    *<p>
    *<pre>{@literal
    * helm upgrade weblogic-operator kubernetes/charts/weblogic-operator
    * --namespace ns-ipqy
    * --reuse-values
    * --set "domainNamespaces={ns-xghr,ns-idir}"
-   * --set "externalRestEnabled=false"
-   * --set "elkIntegrationEnabled=false"
-   * --set "enableClusterRoleBinding=false"
-   * --set "externalRestHttpsPort=0">
    * }
    * </pre>
    * </p>
-   * Test verifies NamespaceWatchingStarted event is logged when operator starts watching an another domain namespace.
    */
   @Order(11)
-  @Test
-  public void testK8SEventsStartWatchingNS() {
+  @ParameterizedTest
+  @ValueSource(booleans = { true, false })
+  public void testK8SEventsStartStopWatchingNS(boolean enableClusterRoleBinding) {
+    logger.info("testing testK8SEventsStartStopWatchingNS with enableClusterRoleBinding={0}",
+        enableClusterRoleBinding);
     OffsetDateTime timestamp = now();
+
     logger.info("Adding a new domain namespace in the operator watch list");
-    upgradeAndVerifyOperator(opNamespace, domainNamespace1, domainNamespace2);
+    List<String> domainNamespaces = new ArrayList<>();
+    domainNamespaces.add(domainNamespace1);
+    domainNamespaces.add(domainNamespace2);
+    opParams = opParams.domainNamespaces(domainNamespaces).enableClusterRoleBinding(enableClusterRoleBinding);
+    upgradeAndVerifyOperator(opNamespace, opParams);
+
     logger.info("verify NamespaceWatchingStarted event is logged");
     checkEvent(opNamespace, domainNamespace2, null, NAMESPACE_WATCHING_STARTED, "Normal", timestamp);
+
+    timestamp = now();
+
+    logger.info("Removing domain namespace in the operator watch list");
```

Comment:
Please indicate which domain space is removed from watch

Reply:
fixed